eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -186,6 +185,20 @@ steps:
     displayName: Build MAUI iOS
     workingDirectory: $(Build.SourcesDirectory)/MauiTesting
 
+  - script: |
+      chmod -R a+r .
+      # remove net6.0-maccatalyst to work around https://github.com/dotnet/sdk/issues/21877
+      cp Microsoft.NetConf2021.Maui.csproj Microsoft.NetConf2021.Maui.csproj.bak
+      sed -i'' -e 's/net6.0-ios;net6.0-maccatalyst/net6.0-ios/g' Microsoft.NetConf2021.Maui.csproj
```

Comment:
I bet we can come up with a sed line that will survive more classes of csproj file edits. I'll pester you in person.

Reply:
Working on this: #67263

Reply:
Will merge this, then copy the sed change to this section.